src/test/skript/tests/syntaxes/effects/EffEquip.sk

```diff
@@ -1,12 +1,42 @@
 test "equip effect":
-	spawn a zombie at spawn of "world":
-		set {_entity} to event-entity
+	spawn a zombie at (spawn of world "world"):
+		set {_z} to entity
 
-	equip {_entity} with a diamond chestplate
-	assert chestplate of {_entity} is a diamond chestplate with "entity was not wearing a diamond chestplate"
-	set chestplate of {_entity} to air
+	assert helmet of {_z} is air with "a normally spawned zombie shouldn't be wearing a helmet"
+	assert chestplate of {_z} is air with "a normally spawned zombie shouldn't be wearing a chestplate"
+	assert leggings of {_z} is air with "a normally spawned zombie shouldn't be wearing leggings"
+	assert boots of {_z} is air with "a normally spawned zombie shouldn't be wearing boots"
```

Comment:
are you sure this is true? naturally spawned zombies have armor occasionally in vanilla.